data_generation/download_tasks.py

```diff
@@ -0,0 +1,66 @@
+"""This file downloads the requested tasks"""
+import os
+
+from absl import app
+from absl import flags
+from absl import logging
+
+import inductiva
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string("path_to_sim_ids", None,
+                    "The path to the simulation ids txt files.")
+
+flags.DEFINE_string(
+    "download_dir", None,
+    "Where to download the data. If None, the data is not downloaded.")
+
+
+def main(_):
+    inductiva.api_key = os.environ["INDUCTIVA_API_KEY"]
+
+    # Read the simulation ids
+    with open(FLAGS.path_to_sim_ids, "r", encoding="utf-8") as f:
+        task_ids = f.read().splitlines()
+    logging.info("Number of tasks: %s", len(task_ids))
+
+    tasks_still_running = []
+    tasks_successfully_completed = []
+    tasks_failed = []
+    tasks_killed = []
+    other_tasks = []
+    for task_id in task_ids:
```

Comment:
I know you want this for two purposes, but I think it would be better to split it into a monitor scripting and a download_script. Or just change the name of the script to mention reflect the possibilities.